### ISSUE TITLE
“动态” Unable to load

### ISSUE THREAD

**Issue by FlyGoat** [edwardez/BangumiN#18]
![Screenshot_20190808-011225_BangumiN](https://user-images.githubusercontent.com/5955297/62643154-e5420000-b979-11e9-8f7d-9042bdd592ce.png)
After login, "动态" tag refused to load with authentication failure warning while other functions like "进度", "讨论" or "主页" work fine.
![Screenshot_20190808-011242_BangumiN](https://user-images.githubusercontent.com/5955297/62643332-5386c280-b97a-11e9-8647-e13ba9be8715.png)

I'm happy to help with debugging like dumping logs or doing tests.

Thanks.

FYI: I don't have any "好友"  on bangumi, so API format may different with others since it will display the timeline of whole site. Meanwhile users with at least one "好友" will only get timeline from their "好友".

**Comment by edwardez** [comment]
Thanks for filing this bug!

Yes, this is an known issue and [here](https://bgm.tv/group/topic/351515#post_1473727) is some additional discussion. 

tl;dr is timeline is obtained by fetching html, and there is no easy way way to check whether a user is successfully authenticated since all BangumiN gets is a raw html -- especially for user who don't add any friends on Bangumi.

This issue should be fixed in the next release.

**Comment by edwardez** [comment]
Hi,

The latestest release 0.4.3 fixes this issue, please take a look and let me know if you have any questions.

https://github.com/edwardez/BangumiN/releases/latest

**Comment by FlyGoat** [comment]
I can confirm it have been resolved.
Thanks a lot.
